www/caddy-custom/Makefile

```
@@ -27,8 +27,9 @@ SUB_FILES=	caddy
 
 PLIST_FILES=	bin/caddy
 
-CADDY_VERSION!=	${MAKE} -C ${.CURDIR}/../caddy  -V '$${DISTVERSION}.$${PORTREVISION}'
-XCADDY_VERSION!=${MAKE} -C ${.CURDIR}/../xcaddy -V '$${DISTVERSION}.$${PORTREVISION}'
+CADDY_VERSION!=	${MAKE} -C ${.CURDIR}/../caddy  -v '$${DISTVERSION}.$${PORTREVISION}'
+XCADDY_VERSION!=${MAKE} -C ${.CURDIR}/../xcaddy -v '$${DISTVERSION}.$${PORTREVISION}'
+CADDY_BUILD!=	${MAKE} -C ${.CURDIR}/../caddy  -v '$${DISTVERSIONPREFIX}$${DISTVERSION}'
```

just for neatness ;)

```suggestion
CADDY_BUILD!=	${MAKE} -C ${.CURDIR}/../caddy  -v '$${DISTVERSIONPREFIX}$${DISTVERSION}'
CADDY_VERSION!=	${MAKE} -C ${.CURDIR}/../caddy  -v '$${DISTVERSION}.$${PORTREVISION}'
XCADDY_VERSION!=${MAKE} -C ${.CURDIR}/../xcaddy -v '$${DISTVERSION}.$${PORTREVISION}'
```